oaknational/Oak-Web-Application
# PUPIL-959 feat(results): add printable results page behind feature flag

## Description

Music year: 2020

- List of changes



## Issue(s)

Fixes #

## How to test

### Acceptance criteria

- [x]  hidden behind a feature flag
- [x]  stores results in local storage using provider
- [x]  clicking opens a new tab with the printable results


on OWA the button should be available; going to https://eu.posthog.com/project/124/feature_flags/32707 you can switch the button on and off by disabling the feature flag

1. Go to https://deploy-preview-2766--oak-web-application.netlify.thenational.academy/pupils/programmes/maths-primary-year-6/units/use-knowledge-of-part-whole-structure-to-solve-additive-problems/lessons/explain-how-a-combination-of-different-parts-can-be-equivalent-to-the-same-whole/review?update_tracking_profile=true
2. Click on "share lesson result" button
3. You should see results page opened in a new tab

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
